qa/integration/specs/logstash_to_logstash_spec.rb

```diff
@@ -25,14 +25,7 @@
 
 describe "Logstash to Logstash communication Integration test" do
 
-  before(:all) {
-    @fixture = Fixture.new(__FILE__)
-    # backup original setting file since we change API port number, and restore after all tests
-    FileUtils.cp(@fixture.get_service('logstash').application_settings_file, "#{@fixture.get_service('logstash').application_settings_file}.original")
-  }
-
```

Comment:
~~~suggestion
  before(:all) {
    @fixture = Fixture.new(__FILE__)
  }
~~~

We still need the `@fixture`